scss/civicrm/common/_modals.scss

```diff
@@ -141,15 +141,23 @@
           margin-left: 0 !important;
         }
 
-        &:hover {
+        &:hover,
+        &:active,
+        &:focus {
           color: $crm-white;
-          opacity: .8;
+          opacity: .8 !important;
```

Comment:
why the !important?

Reply:
the button extends the %btn-civi which has opacity: 1 !important, to override that we need important here.